llvm/include/llvm/IR/IntrinsicsAMDGPU.td

```diff
@@ -1368,6 +1391,28 @@ def int_amdgcn_struct_ptr_buffer_atomic_cmpswap : Intrinsic<
 // gfx908 intrinsic
 def int_amdgcn_struct_buffer_atomic_fadd : AMDGPUStructBufferAtomic<llvm_anyfloat_ty>;
 def int_amdgcn_struct_ptr_buffer_atomic_fadd : AMDGPUStructPtrBufferAtomic<llvm_anyfloat_ty>;
+// gfx12 intrinsic
+def int_amdgcn_struct_buffer_atomic_fadd_v2bf16 : Intrinsic <
+  [llvm_v2i16_ty],
```

Comment:
Probably should start using v2bf16 for these, although we have the bad precedent of not using it for the existing operations (I'm currently working on fixing this for some other intrinsics)

Reply:
Patch to v2bf16 legal is committed, you can now update this to use it

Reply:
Done